openlibrary/templates/lib/header_dropdown.html

```diff
@@ -57,8 +57,9 @@
               </li>
             $else:
               <li>
-              $if 'post' in link:
-                <form action="$(link['post'])" method="post">
+              $if 'name' in link:
+                $ name = 'name=%s' % link['name']
```

Comment:
```suggestion
              $if 'post' in link:
                $ name = 'name=%s' % link['name'] if 'name' in link else ''
```
If we execute this block, and `post` is not in `link`, every page will display an error message.  Please use the suggested code, above.

Reply:
done!